src/extension.ts

```diff
@@ -152,6 +153,9 @@ export async function activate(context: vscode.ExtensionContext) {
   octokit = await credentials.getOctokit();
 
   vscode.window.onDidChangeTextEditorSelection(async (selection) => {
+    let churnResult = getFileChurn(selection.textEditor.document.uri.fsPath);
```

Comment:
We are doing this for the user _every_ time they highlight?
This probably will impact the whole editor's performance.